test/Polly.Extensions.Tests/Telemetry/TelemetryOptionsTests.cs

```diff
@@ -21,4 +23,29 @@ public void Ctor_EnsureDefaults()
         using var response = new HttpResponseMessage(HttpStatusCode.OK);
         options.ResultFormatter(resilienceContext, response).Should().Be(200);
     }
+
+    [Fact]
+    public void CopyCtor_Ok()
```

Comment:
Is it worth us adding an assert that validates how many properties `TelemetryOptions` has so that if we add more in future a test fails to act as a reminder to ensure the copy constructor is implemented correctly?

I've been hit my this sort of thing recently 😄: https://github.com/domaindrivendev/Swashbuckle.AspNetCore/pull/2796

Reply:
Done.